indico/web/client/js/react/components/pictures/PictureArea.jsx

```diff
@@ -47,8 +47,14 @@ export function PictureArea({
     : Translate.string('Take a picture');
   const dropzoneDisabled = openUploadDialog === null;
 
+  // Delete the `role` and `tabIndex` props as the drag area nests
+  // interactive elements. Nesting interactive elements within other
+  // interactive elements is illegal.
+  const dragAreaProps = getRootProps();
+  const {role: _1, tabIndex: _2, ...dragAreaPropsWithoutRole} = dragAreaProps;
```

Comment:
IMHO `const dragAreaProps = _.omit(getRootProps(), ['role', 'tabIndex']);` would be much more readable